### PR TITLE
fix INSIGHT-921 codeStyle

### PR DESCRIPTION
**Pull request by jcarsique**
- style(INSIGHT-921): code format Java
- style(INSIGHT-921): code format (other types)

Rules are shared in nuxeo/nuxeo-ai-tools#6

### CODE REVIEW

File: addons/nuxeo-ai-gcp-core/README.md [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
-  * gcp.objectLocalizer - Calls GCP to detect objects, and their bounding boxes in images
-  
+
+* gcp.imageLabels - Calls GCP for object detection in images to produce labels
+* gcp.textDetection - Calls GCP to detect text, and its bounding boxes in images
+* gcp.faceDetection - Calls GCP to detect faces, and their bounding boxes in images
+* gcp.logoDetection - Calls GCP to detect logos, and their bounding boxes in images
+* gcp.landmarkDetection - Calls GCP to detect landmarks in images
+* gcp.objectLocalizer - Calls GCP to detect objects, and their bounding boxes in images
+
```
Comment (andreinechaev) [inline addons/nuxeo-ai-gcp-core/README.md:11]: funny that it didn't touch this one

**Review by andreinechaev — APPROVED** [review]

File: addons/README.md [hunk 2]
```diff
@@ -1,25 +1,40 @@
...
+* [nuxeo-ai-image-quality](https://github.com/nuxeo/nuxeo-ai/tree/master/addons/nuxeo-ai-image-quality-core#nuxeo-ai-image-quality)
+  - Enrichment services that uses [Sightengine](https://sightengine.com/).
+* [nuxeo-ai-aws](https://github.com/nuxeo/nuxeo-ai/blob/master/addons/nuxeo-ai-aws-core/README.md#nuxeo-ai-aws-integration)
+  - Enrichment services that use [Amazon Web Services](https://aws.amazon.com).
+* [nuxeo-ai-image-quality](https://github.com/nuxeo/nuxeo-ai/tree/master/addons/nuxeo-ai-image-quality-core#nuxeo-ai-image-quality)
+  - Enrichment services that uses [Sightengine](https://sightengine.com/).
+* [nuxeo-ai-aws](https://github.com/nuxeo/nuxeo-ai/blob/master/addons/nuxeo-ai-aws-core/README.md#nuxeo-ai-aws-integration)
+  - Enrichment services that use [Amazon Web Services](https://aws.amazon.com).
+* [nuxeo-ai-gcp](https://github.com/nuxeo/nuxeo-ai/blob/master/addons/nuxeo-ai-gcp-core/README.md) - Enrichment services
+  that use [Google Vision](https://cloud.google.com/vision/).
```
[the remark this answers is not in the document]
Comment (jcarsique) [reply addons/README.md:14]: Indeed, it is :/
https://github.github.com/gfm/#lists

[the remark this answers is not in the document]
Comment (jcarsique) [reply addons/README.md:14]: It's a random side effect of the margin wrapping: the wrapping happens asap after the link... on the hyphen! that is then interpreted as a list item.

File: addons/nuxeo-ai-gcp-core/README.md (hunk 1 above)
Comment (jcarsique) [reply addons/nuxeo-ai-gcp-core/README.md:11]: because it didn't exceed the margin

**Pull request merged**